src/components/modules/banners/contributions/ContributionsBannerCommonStyles.ts

```diff
@@ -0,0 +1,50 @@
+import { css } from '@emotion/core';
```

Comment:
This replaces `ContributionsBannerStyles.ts`. It now only contains styles that are shared by components